feat: add novu cli dev command flags (#722)
novuhq/docs: workflow/studio.mdx

```bash
npx novu@latest dev
```

> <Info>Learn how to what [flag](#novu-cli-flags) to use for customization</Info>

Reviewer: Improving the sentence readability

```suggestion
<Info>Learn how to use the `novu` CLI package and the available [CLI flags](#novu-cli-flags) to use for customization</Info>
```

- **YOUR_SERVER_PORT** - This accepts the port number where your server is running. Defaults to 4000.
- **YOUR_NOVU_ROUTE_PATH** - This is the mounted path of the framework `serve` function. Defaults to `/api/novu`.

## Novu CLI flags

The Novu CLI command `npx novu@latest dev` supports a number of flags:

| Flag | Long form usage example   | Description                   | Default value               |
| ---- | -----------------------   | ----------------------------- | --------------------------- |
| -p   | --port `<port>`           | Local bridge application port | 4000                        |
| -r   | --route `<route>`         | Bridge application route      | /api/novu                   |
| -o   | --origin `<origin>`       | Bridge application origin     | http://localhost            |
| -d   | --dashboard-url `<url>`   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
| -sp  | --studio-port `<port>`    | Local Studio server port      | 2022                        |

> | -t   | --tunnel `<url>`          | Custom tunnel url             | null              |

Reviewer: ```suggestion
| -t   | --tunnel `<url>`          | Custom tunnel url             | null                       |
```

> | -t   | --tunnel `<url>`          | Custom tunnel url             | null                       |

Reviewer: ```suggestion
| -t   | --tunnel `<url>`          | Custom tunnel url             | null                        |
```

> | flag | long form usage example   | description                   | default value               |
> | ---- | -----------------------   | ----------------------------- | --------------------------- |
> | -p   | --port `<port>`           | local bridge application port | 4000                        |
> | -r   | --route `<route>`         | bridge application route      | /api/novu                   |
> | -o   | --origin `<origin>`       | bridge application origin     | http://localhost            |
> | -d   | --dashboard-url `<url>`   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
> | -sp  | --studio-port `<port>`    | Local Studio server port      | 2022                        |
> | -t   | --tunnel `<url>`          | self hosted tunnel url        | https://my-tunnel.ngrok.app |

Reviewer: Using uppercase consistently, and fixing fixing the tunnel default (it doesn't have one). Also, it's not a self-hosted tunnel (other platforms host it), it's just a custom tunnel.

```suggestion
| Flag | Long form usage example   | Description                   | Default value               |
| ---- | -----------------------   | ----------------------------- | --------------------------- |
| -p   | --port `<port>`           | Local bridge application port | 4000                        |
| -r   | --route `<route>`         | Bridge application route      | /api/novu                   |
| -o   | --origin `<origin>`       | Bridge application origin     | http://localhost            |
| -d   | --dashboard-url `<url>`   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
| -sp  | --studio-port `<port>`    | Local Studio server port      | 2022                        |
| -t   | --tunnel `<url>`          | Custom tunnel url        | null |
```
